core-services/image-pruner/admin_rbac.yaml

```diff
@@ -0,0 +1,18 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: image-pruner
+    namespace: ci
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
```

Comment:
I the current clusterrolebinding there is also:
```
userNames:
- system:serviceaccount:ci:image-pruner
```
I am not sure if that will be added automatically, but it worth taking a look.

Reply:
Yeah, I wondered about that too. I have found nothing about that on https://kubernetes.io/docs/reference/access-authn-authz/rbac/ or https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#clusterrolebinding-v1-rbac-authorization-k8s-io, so it really looks like something that will be put there automatically
/shrug

Reply:
Yeah it gets generated